<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    
    <meta name="description" content="StaPH-B Training Website">
    <meta name="keywords" content="bioinformatics, public health, infectious disease">

    <meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Glossary of Bioinformatic Terms"/>
<meta name="twitter:description" content="This glossary is adapted from Kelsey Florek and her Midwest Region AMD Pages on GitHub. The glossary is reproduced here for the benefit of those who cannot access GitHub.
Bioinformatics This discipline is involved in the collection, analysis, and storage of complex biological data, including, but not limited to genetic data. The field combines computer science, statistics, and several aspects of biology to be able to understand highly complex data."/>

    <meta property="og:title" content="Glossary of Bioinformatic Terms" />
<meta property="og:description" content="This glossary is adapted from Kelsey Florek and her Midwest Region AMD Pages on GitHub. The glossary is reproduced here for the benefit of those who cannot access GitHub.
Bioinformatics This discipline is involved in the collection, analysis, and storage of complex biological data, including, but not limited to genetic data. The field combines computer science, statistics, and several aspects of biology to be able to understand highly complex data." />
<meta property="og:type" content="article" />
<meta property="og:url" content="http://www.staphb.org/resources/glossary/" />
<meta property="article:published_time" content="2019-12-18T00:44:23+00:00" />
<meta property="article:modified_time" content="2019-12-18T00:44:23+00:00" />


    
      <base href="http://www.staphb.org/resources/glossary/">
    
    <title>
  Glossary of Bioinformatic Terms · StaPH-B
</title>

    
      <link rel="canonical" href="http://www.staphb.org/resources/glossary/">
    

    <link href="https://fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather:300,700%7CSource+Code+Pro:400,700" rel="stylesheet">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css" integrity="sha384-KA6wR/X5RY4zFAHpv/CnoG2UW1uogYfdnP67Uv7eULvTveboZJg0qUpmJZb5VqzN" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />

    
      
      
      <link rel="stylesheet" href="http://www.staphb.org/css/coder.min.28d751104f30c16da1aa1bb04015cbe662cacfe0d1b01af4f2240ad58580069c.css" integrity="sha256-KNdREE8wwW2hqhuwQBXL5mLKz&#43;DRsBr08iQK1YWABpw=" crossorigin="anonymous" media="screen" />
    

    

    

    
      <link rel="stylesheet" href="http://www.staphb.org/css/custom.css" />
    

    
    
    <link rel="icon" type="image/png" href="http://www.staphb.org/img/favicon.ico" sizes="32x32">
    <link rel="icon" type="image/png" href="http://www.staphb.org/img/favicon.ico" sizes="16x16">

    <meta name="generator" content="Hugo 0.62.2" />
  </head>

  
  
  <body class="colorscheme-light">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="http://www.staphb.org/">
      StaPH-B
    </a>
    <input type="checkbox" id="menu-toggle" />
    <label class="menu-button float-right" for="menu-toggle"><i class="fas fa-bars"></i></label>
    <ul class="navigation-list">
      
        
          <li class="navigation-item">
            <a class="navigation-link" href="http://www.staphb.org/posts/">Blog</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="http://www.staphb.org/projects/">Projects</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="http://www.staphb.org/resources/">Resources</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="http://www.staphb.org/training/">Training</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="http://www.staphb.org/about/">About</a>
          </li>
        
          <li class="navigation-item">
            <a class="navigation-link" href="http://www.staphb.org/history/">History</a>
          </li>
        
      
      
    </ul>
  </section>
</nav>


      <div class="content">
        
  <section class="container page">
  <article>
    <header>
      <h1>Glossary of Bioinformatic Terms</h1>
    </header>

    <p>This glossary is adapted from Kelsey Florek and her <a href="https://staph-b.github.io/midwest-region/">Midwest Region AMD Pages</a> on GitHub.  The glossary is reproduced here for the benefit of those who cannot access GitHub.</p>
<h5 id="bioinformatics">Bioinformatics</h5>
<p>This discipline is involved in the collection, analysis, and storage of complex biological data, including, but not limited to genetic data. The field combines computer science, statistics, and several aspects of biology to be able to understand highly complex data.</p>
<h5 id="amd-advanced-molecular-detection">AMD (Advanced Molecular Detection)</h5>
<p>This commonly may mean any form of sequencing or high-level molecular technology. There is an Office of Advanced Molecular Detection (OAMD) at CDC that provides support and input on many cross-cutting areas using advanced technologies</p>
<h4 id="genomics">Genomics</h4>
<hr>
<ul>
<li>
<p><strong>Gene</strong> – sequence of DNA that encodes a region for a molecule (ie protein or enzymes) with function</p>
</li>
<li>
<p><strong>Genotype vs Phenotype</strong> – <strong>Genotypes</strong> are the genetic make-up of an individual. <strong>Phenotypes</strong> are the physical traits and characteristics of an individual and are influenced by their genotype and the environment.</p>
</li>
<li>
<p><strong>Gene Regulation</strong></p>
<ul>
<li><strong>Cis regulation</strong> – Any molecular interaction that regulates the transcription of nearby genes on the same DNA molecule, such as the role of a gene promote</li>
<li><strong>Trans regulation</strong> – Any molecular interaction that regulates the transcription of genes on a different DNA molecule, such as a transcription factor regulating both alleles of a target gene or genes.</li>
</ul>
</li>
<li>
<p><strong>Indels</strong> – A type of DNA sequence variation marked by the insertion or deletion of nucleotides, Defined as insertion or deletion of bases in the genome of an organism. An INDEL mutation named with the blend of insertion and deletion. It refers to a length difference between two allele caused by a SEQUENCE INSERTION or by a SEQUENCE DELETION.</p>
</li>
<li>
<p><strong>Mobile elements</strong> – sequences of DNA that have the ability to move from one organism to another, move into different locations in a genome or change the copy number in the bacteria. These element include plasmids, bacteriophages and transposable elements</p>
</li>
<li>
<p><strong>Mutation Rates</strong> – rate in which a single nucleotide is replaced by a different nucleotide
Recombination – rearrangement of genetic material. Both prokaryotes and eukaryotes can perform recombination.</p>
</li>
<li>
<p><strong>Operon</strong> – It is a genetic regulatory system found in bacteria and their viruses in which genes coding for functionally related proteins are clustered along the DNA. This feature allows protein synthesis to be controlled coordinately in response to the needs of the cell. A typical operon consists of a group of structural genes that code for enzymes involved in a metabolic pathway, such as the biosynthesis of an amino acid. These genes are located contiguously on a stretch of DNA and are under the control of one promoter (a short segment of DNA to which the RNA polymerase binds to initiate transcription). A single unit of messenger RNA (mRNA) is transcribed from the operon and is subsequently translated into separate proteins.</p>
</li>
<li>
<p><strong>Structural variants (SVs)</strong> – DNA sequence variants that are 50bp or larger, including insertions, deletions, inversions, duplications and translocations</p>
</li>
</ul>
<h4 id="programming-languages">Programming Languages</h4>
<hr>
<ul>
<li>
<p><strong>BASH / Shell Script</strong> – A programming language that is directly used by the Linux operating system. It allows for direct control of the operating system functions.</p>
</li>
<li>
<p><strong>Command line / Command based</strong> – The command line allows the user to interact with their computer without graphical user interface (GUI) or mouse. The command line is essential to perform bioinformatics analysis. It allows the user to manipulate files and programs in ways that are not possible using a GUI.</p>
</li>
<li>
<p><strong>Linux</strong> – An operating system much like Windows or OSX. The field of bioinformatics relies heavily on Linux-based computers and software. Although some bioinformatics programs can be compiled to run on Mac OS X and Windows systems, many require the Linux environment to run properly.</p>
</li>
<li>
<p><strong>PERL</strong> – Another programming language similar to Python.</p>
</li>
<li>
<p><strong>Python</strong> – Python is a popular programming language object often used for scientific computing. The Biopython utilize python-based software for bioinformatics research, thereby creating high quality reusable modules and scripts.</p>
</li>
</ul>
<h4 id="sequencing-instruments-and-chemistry">Sequencing Instruments and Chemistry</h4>
<hr>
<ul>
<li>
<p><strong>PacBio</strong> –  It is a single molecule real time (SMRT) sequencing developed by Pacific Biosciences, also referred as PacBio sequencing. Unlike second generation sequencing, PacBio sequences each base-pair individually. Currently, this method is widely used to study larger genomes, transcriptome metagenomics and epigenetics.</p>
</li>
<li>
<p><strong>Tape station</strong> – The Agilent 2100 Bioanalyzer (Tape Station) is a microfluidics-based platform for size determination, quantification and quality control of DNA and RNA.</p>
</li>
<li>
<p><strong>Pyrosequencing</strong> –  In pyrosequencing, the sequencing reaction is monitored through the release of the pyrophosphate during nucleotide incorporation. A single nucleotide is added to the sequencing chip which will lead to its incorporation in a template dependent manner. This incorporation will result in the release of pyrophosphate which is used in a series of chemical reactions resulting in the generation of light. Light emission is detected by a camera which records the appropriate sequence of the cluster. Any unincorporated bases are degraded by apyrase before the addition of the next nucleotide. This cycle continues until the sequencing reaction is complete.</p>
</li>
<li>
<p><strong>Paired end sequencing</strong> –
<img src="http://www.staphb.org/images/paired_end.png" alt="Paired_End"></p>
</li>
</ul>
<p>It involves sequencing both ends of the DNA fragment and generate high quality sequence data. It’s a simple work flow allows generation of unique ranges of insert sizes.</p>
<ul>
<li>
<p><strong>Long read sequencing</strong> – Third generation sequencing also known as long read sequencing (LRS). LRS allows for the retrieval of much longer (&gt;10,000bp) sequencing reads than widely-used short read sequencing systems (75-300bp). Long-read sequencing (LRS) has become increasingly important due to its strengths in resolving complex DNA regions as well as in determining full-length RNA molecules. Two important LRS technologies have been developed during the past few years, including single-molecule, real-time sequencing by Pacific Biosciences, and nanopore sequencing by Oxford Nanopore Technologies.</p>
</li>
<li>
<p><strong>Short read sequencing</strong> – Next generation sequencing also known as short read sequencing. The technology utilizes highest output platform continue to have relatively short read lengths on the order of 35-300 bases per read. The amazing increase in data output and decrease in cost per base sequenced has been driven primarily short-read sequencing technologies such as the Illumina and Ion Torrent platforms.</p>
</li>
<li>
<p><strong>Massively parallel sequencing</strong> – Massively parallel sequencing (MPS) technology, also referred to as next-generation sequencing, is a high-throughput approach to DNA sequencing in which miniaturized, parallelized platforms are used for sequencing 1 million to 43 billion short reads per instrument run. Next-generation sequencing (NGS) is transforming the landscape of clinical microbiology and public health laboratories. The applications of NGS are wide-ranging and include whole-genome sequencing, microbiome analysis/metagenomics, transcriptome profiling, infectious disease diagnosis, pathogen discovery, and public health surveillance.</p>
</li>
<li>
<p><strong>Base Calling</strong> – Base calling is the process of assigning nucleotides or bases (ATGC) with chromatogram peaks. Next generation sequencing platforms, which use fluorescently labeled reversible terminators have a unique color for each base. These are incorporated in to the complementary strand of the DNA template and captured with a sensitive CCD camera. These images are processed into signals which are used to infer the order of nucleotides also referred as base calling. The accuracy is, measured by a Q score (Phred quality score), a common metric to assess the accuracy of a sequencing run.</p>
</li>
<li>
<p><strong>Adapter</strong> – They are oligonucleotides bound to 5’ and 3’ end of each fragment in a sequencing library. Adapters include platform-specific sequences for fragment recognition by the sequencer. For example, the P5 and P7 sequences enable library fragments to bind to the flow cells of Illumina platforms.</p>
</li>
<li>
<p><strong>Barcode sequences</strong> – Multiplex sequencing allows large numbers of libraries to be pooled and sequenced simultaneously during a single run on a high-throughput Illumina NextGen sequencer. Individual &ldquo;barcode&rdquo; sequences are added to each DNA fragment during next-generation sequencing (NGS) library preparation so that each read can be identified and sorted out based on the fingerprinting of the genome prior to final data analysis.</p>
</li>
<li>
<p><strong>Sanger Sequencing</strong> – DNA sequencing method that relies on a primer to bind to the DNA sequence of interest to amplify the region of interest, sequences a single DNA fragment</p>
</li>
</ul>
<h4 id="bioinformatic-analysis">Bioinformatic Analysis</h4>
<hr>
<ul>
<li>
<p><strong>ANI (average nucleotide identity)</strong> – provides a measure of how similar two genomes are at the nucleotide level</p>
</li>
<li>
<p><strong>Coverage</strong> – number of reads that are aligned at a specific nucleotide</p>
</li>
<li>
<p><strong>Contig (contiguous DNA)</strong> – sequences of DNA that have been assembled together as the result of overlapping sequences into a consensus sequence and resulting in a contiguous sequence</p>
</li>
<li>
<p><em><strong>De novo</strong></em> <strong>assembly</strong> – assembly of sequencing reads without using a reference sequence as a guide</p>
</li>
<li>
<p><strong>DNA Sequence Analysis</strong></p>
<ul>
<li>
<p><strong>CgMLST (Core genome MLST)</strong> – examines only the genes that are shared among all of the strains to assign alleles, these genes are shared by the majority of the strains within a genus</p>
</li>
<li>
<p><strong>MLST (Multi Locus Sequence Typing)</strong> – traditionally used 6-12 housekeeping genes and Sanger sequencing to identify gene variants and alleles, which can be used to generate sequence types (ST), standardized allele schemes</p>
</li>
<li>
<p><strong>WgMLST (whole genome MLST)</strong> – examines all genes within the genome to assign allele codes to an organism, examined on a gene by gene basis and can be either the whole gene or part of the gene to form the locus, variants within the locus are given new allele codes to generate wgMLST types.</p>
</li>
<li>
<p><strong>Consensus</strong> – sequence of nucleotides that represents the most commonly found base pair at a specific position, identified by aligning the raw reads
Reference assembly: mapping reads to a reference genome in order to assembly the reads into contigs or closed genomes</p>
</li>
<li>
<p><strong>Reference genome</strong> – a consensus sequence of DNA for an organism</p>
</li>
<li>
<p><strong>Variants</strong> – sequences of DNA that differ at specific positions when the sequences are aligned</p>
</li>
<li>
<p><strong>Alleles</strong> – variant of a sequence, variant does not have to affect the phenotype</p>
</li>
<li>
<p><strong>SNP (single nucleotide polymorphism)</strong> – a nucleotide position that can be used to discriminate strains in a population</p>
</li>
<li>
<p><strong>HqSNP (High quality SNP)</strong> – utilizes a program such a lyve-SET to identify SNPs that have at least 20X coverage and uses a high quality reference genome</p>
</li>
<li>
<p><strong>Query sequence</strong> –  BLAST (Basic Local Alignment Search Tool) uses statistical methods to compare a DNA or protein input sequence (“query sequence”) to a database of sequences (“subject sequences”) and return those sequences that have a significant level of similarity to the query sequence.</p>
</li>
</ul>
</li>
<li>
<p><strong>File Types</strong></p>
<ul>
<li>
<p><strong>FASTQ file</strong> – contains the raw sequencing data, contains the NGS read data including both the DNA sequences and the quality sequencing information for each read, each read has the read ID, the base calls, additional information and sequencing quality metrics</p>
</li>
<li>
<p><strong>FASTA file</strong> – contains DNA/RNA or protein sequences, “&gt;” denotes the header/sequence identifier</p>
</li>
<li>
<p><strong>SAM (Sequence Alignment/Map) file</strong> – contains the alignment of reads to a reference genome</p>
</li>
<li>
<p><strong>BAM file</strong> – compressed binary version of a SAM file</p>
</li>
</ul>
</li>
<li>
<p><strong>Filtering</strong> – Pre processing of raw reads before assembly to remove any reads that do not meet a specified quality parameters</p>
</li>
<li>
<p><strong>Gene or sequence annotation</strong> – Sequence annotation is the process of marking specific regions or sites of interest in the protein sequence, such as post-translational modifications, binding sites, enzyme active sites, local secondary etc. Annotation may be defined as the part of genome analysis that is customarily performed before a genome sequence is deposited in GenBank and described in a published paper.</p>
</li>
<li>
<p><strong>k-mer</strong> – k-mer are often used in computational genomics. k-mer refer to all the possible sub-sequences of a defined length obtained from a set of sequence information.</p>
</li>
<li>
<p><strong>Metagenomics</strong> – the study of complex bacterial communities to examine diversity of organisms at various taxonomic levels. Sequencing can be performed using 16S primers (16S sequencing) or by sequencing all DNA in a sample (shotgun sequencing) to identify genes present and identification of organisms</p>
</li>
<li>
<p><strong>Multiple sequence alignment</strong> – Multiple Sequence Alignment (MSA) is generally the alignment of three or more biological sequences (protein or nucleic acid) of similar length. In contrast, Pairwise Sequence Alignment tools are used to identify regions of similarity that may indicate functional, structural and/or evolutionary relationships between two biological sequences. ClustalW is widely used program for multiple sequence alignment.</p>
</li>
<li>
<p><strong>NCBI (National Center for Biotechnology Information)</strong> – part of the National Laboratory of Medicine and National Institute of health. The Entrez Global Query Cross Database search system is widely adopted by NCBI for array of nucleotide and protein sequences, protein structures, PubMed, taxonomy, whole genome etc.</p>
</li>
<li>
<p><strong>N50</strong> – A weighted average length; specifically, the N50 length is the length such that 50% of the genome has been assembled into contig or scaffold sequences of this length or longer.</p>
</li>
<li>
<p><strong>Pipeline</strong> – the chain of processing data and moving files through different processes or functions to produce the desired files and results, also referred to as a workflow</p>
</li>
<li>
<p><strong>Proteomics</strong> – the study of all the proteins in a system and their interactions, structure and composition</p>
</li>
<li>
<p><strong>Q Score</strong> – the quality score is the probability of an incorrect base call; Q30: 1 in 1000 times incorrect, thus, each nucleotide is 99.9% accurate</p>
</li>
<li>
<p><strong>Reading frame and Open reading frame (ORF)</strong> –  An ORF is a sequence of DNA that starts with start codon “ATG” and ends with any of the three termination codons (TAA, TAG, TGA). Depending on the starting point, there are six possible ways (three on forward strand and three on complementary strand) of translating any nucleotide sequence into amino acid sequence according to the genetic code. These are called reading frames.</p>
</li>
<li>
<p><strong>Reads</strong> – fragments of sequenced DNA base pairs that are generated during sequencing, the raw data from a sequencing machine</p>
</li>
<li>
<p><strong>Sequence Similarity</strong> – It is a method of searching sequence databases by using alignment to a query sequence. Similarity of two sequences refers to the degree of match between corresponding positions in sequence.</p>
</li>
<li>
<p><strong>Source Control / Version Control</strong> – tracking of changes made to files or authorship to be able to recall previous versions. Github is a way to keep track of changes made to files and identify who made the changes.</p>
</li>
</ul>

  </article>
</section>


      </div>

      <footer class="footer">
  <section class="container">
    <p style="font-size:75%;" >Site supported by <a href="http://www.theiagen.com">Theiagen Consulting LLC</a></p>
  </section>
</footer>

    </main>

    

  </body>

</html>
